azure-pipelines.yml

```diff
@@ -43,42 +43,66 @@ jobs:
       export PATH="/home/docker/.cargo/bin:$PATH"
       export RUSTUP_HOME=/home/docker/.rustup
       export CARGO_HOME=$AGENT_TEMPDIRECTORY/.cargo
-      ./scripts/run_ci_checks.sh fmt
+      export RUSTFLAGS="-D warnings"
+      export RUSTDOCFLAGS="-D warnings"
+      cargo fmt --check
     displayName: 'cargo fmt --check'
   
   - script: |
       export PATH="/home/docker/.cargo/bin:$PATH"
       export RUSTUP_HOME=/home/docker/.rustup
       export CARGO_HOME=$AGENT_TEMPDIRECTORY/.cargo
-      ./scripts/run_ci_checks.sh check
+      export RUSTFLAGS="-D warnings"
+      export RUSTDOCFLAGS="-D warnings"
+      # Soon to be superceded by the commented out `cargo clippy` below.
```

Comment:
Why not simply say something like:

```
 # Cargo check step differs from cargo build step since we can use `--all-features` here.
 # We plan to replace `cargo check` with `cargo clippy` (while keeping flags the same). 
 cargo check --tests --all-features
```

Reply:
Fixed in https://github.com/immunant/c2rust/commit/5bb01cb32657d0ca1cdadd0a6bbe4866b885b028 (on `master`).

Reply:
This is not what I suggested. I suggested you improve the comment in this script because it (and not `run_ci_checks.sh`) is the source of truth w.r.t. what is happening in azure pipelines.

Reply:
Oh, oops, I meant to rebase that change in here so it's updated in both places.  I'll fix that.

Reply:
Rebased correctly now.